src/py/kaleido/_page_generator.py

```diff
@@ -10,6 +13,14 @@
 KJS_PATH = Path(__file__).resolve().parent / "vendor" / "kaleido_scopes.js"
 
 
+def _ensure_path(path: Path | str):
+    _logger.debug(f"Ensuring path {path!s}")
+    if not urlparse(str(path)).scheme.startswith("file"):  # is url
+        return
+    if not Path(path).exists():
+        raise ValueError(f"{path!s} does not seem to be a valid path.")
```

Comment:
more assertive wording

```suggestion
        raise ValueError(f"{path!s} does not exist.")
```